homeassistant/components/rfxtrx/config_flow.py

```diff
@@ -0,0 +1,22 @@
+"""Config flow for RFXCOM RFXtrx integration."""
+import logging
+
+from homeassistant import config_entries
+
+from . import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for RFXCOM RFXtrx."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
```

Comment:
Doesn't the callback make this a push update model?

Reply:
Yea true that